src/libraries/restore/runtime/runtime.depproj

```diff
@@ -50,7 +50,12 @@
       <UseHardlink Condition="'$(_runtimeOSFamily)' == 'FreeBSD'">false</UseHardlink>
     </PropertyGroup>
 
-    <ItemGroup>
+    <ItemGroup Condition="'$(RuntimeFlavor)' == 'Mono'">
+      <HostFxFile Include="@(StoreTestHostFiles)" Condition="'%(StoreTestHostFiles.Filename)' == '$(HostFxrFileName)'" />
```

Comment:
Isn't this going to include `dotnet` and `hostfxr` in the runtime packs?

Reply:
Yes, that probably should be excluded.  @mdh1418 You can look in your shipping folder to validate after you exclude.

Reply:
I commented incorrectly the last time, so trying again ;-).  

We won't be including `dotnet` and `hostfxr` because we are excluding them from `ReferenceLocalCopyPaths` and putting them into `StoreTestHostFiles`.  We want to do this because desktop mono is in a state where we need in tree runtime packs, but also need to make sure testhost keeps working (for now).

Reply:
Last time that I ran the builds for mobile/mono both `dotnet` and `hostfxr` were excluded from lib-runtime-packs. When building for mono, `dotnet` and `hostfxr` existed in `testhost`. When building for mobile, neither were in `testhost`.

When using your suggested changes, the same functionality was achieved.

Reply:
Ah right. I somehow got confused by the fact that we were copying them over to the test host :smile: